indico/modules/events/registration/forms.py

```diff
@@ -610,6 +612,34 @@ def validate_retention_period(self, field):
                                     'that of individual fields.'))
 
 
+class RegistrationBasePriceForm(IndicoForm):
+    action = SelectField(_('Action'), [DataRequired()])
+    base_price = DecimalField(_('Registration fee'),
+                              [NumberRange(min=Decimal('0.01'), max=999999.99), HiddenUnless('action', 'custom'),
```

Comment:
Can you match the value we use on the model (`999999999.99`)?

See https://talk.getindico.io/t/i-cannot-set-the-registration-fee-larger-than-999999-99/3387/2 (needs to be fixed when editing the base price as well)

Reply:
I'd do it in a separate PR, not related to this change here. feel free to open one if you want!